docs/iris/src/userguide/index.rst

```diff
@@ -30,6 +30,7 @@ User guide table of contents
    saving_iris_cubes.rst
    navigating_a_cube.rst
    subsetting_a_cube.rst
+   real_and_lazy_data.rst
```

Comment:
I don't know at what point this disappeared... This needs to be here so that the new section appears in the userguide contents and I'm sure I added it in before 😕